SConstruct

```diff
@@ -74,6 +77,53 @@ def AssertG4Version(path):
             Abort(msg)
 
 
+## write nexus-config file
+def WriteNexusConfig(dir_prefix):
```

Comment:
I was trying to find out if SCons provides any standard tool to generate such configuration scripts, but the documentation doesn't mention it at all.